StripePayments/StripePayments/Source/API Bindings/STPAPIClient+Payments.swift

```diff
@@ -920,7 +920,7 @@ extension STPAPIClient {
         )
 
         var params = [
-            "app": String(data: appData ?? Data(), encoding: .utf8) ?? "",
+            "app": String(decoding: appData ?? Data(), as: UTF8.self),
```

Comment:
I've always used `String(data:`.. curious what the change here does?

Reply:
The new initializer is non-optional. The linter made do this 🤷. Maybe I have a different version from CI :(